src/ceph-disk/ceph_disk/main.py

```diff
@@ -665,6 +665,19 @@ def get_dev_size(dev, size='megabytes'):
         os.close(fd)
 
 
+def is_diskdevice
+    dev = os.path.realpath(dev)
```

Comment:
this must reside outside of the function: it matters to the caller and makes a difference when it is gone